src/avs/AVSList.jsx

```diff
@@ -199,55 +196,55 @@ export default function AVSList() {
           </TableHeader>
           <TableBody
             emptyContent={
-              <div className="flex h-[41.8rem] flex-col items-center justify-center">
+              <div className="flex flex-col items-center justify-center">
                 <span className="text-lg text-foreground-2">
-                  No result found for {truncate(state.searchTerm ?? '')}
+                  No result found for {state.searchTerm?.substring(0, 42)}...
                 </span>
               </div>
             }
           >
             {state.isFetchingAvsData
               ? [...Array(10)].map((_, i) => (
                   <TableRow key={i} className="border-t border-outline">
-                    <TableCell className="h-[3.82rem] w-2/5">
-                      <Skeleton className="h-5 rounded-md dark:bg-default" />
+                    <TableCell className="w-2/5 px-5 py-6">
+                      <Skeleton className="h-5 rounded-md bg-default" />
                     </TableCell>
-                    <TableCell className="w-1/5">
-                      <Skeleton className="h-5 rounded-md bg-default dark:bg-default" />
+                    <TableCell className="w-1/5 px-5 py-6">
+                      <Skeleton className="h-5 rounded-md bg-default" />
                     </TableCell>
-                    <TableCell className="w-1/5">
-                      <Skeleton className="h-5 rounded-md bg-default dark:bg-default" />
+                    <TableCell className="w-1/5 px-5 py-6">
+                      <Skeleton className="h-5 rounded-md bg-default" />
                     </TableCell>
-                    <TableCell className="w-1/5">
-                      <Skeleton className="h-5 rounded-md bg-default dark:bg-default" />
+                    <TableCell className="w-1/5 px-5 py-6">
+                      <Skeleton className="h-5 rounded-md bg-default" />
                     </TableCell>
                   </TableRow>
                 ))
               : state.avs?.map((avs, i) => (
                   <TableRow
                     onClick={() =>
-                      navigate(`/avs/${avs.address}`, { state: { avs: avs } })
+                      navigate(`/avs/${avs.address}`, { state: { avs } })
                     }
                     key={`avs-item-${i}`}
-                    className="cursor-pointer border-t border-outline hover:bg-default"
+                    className="cursor-pointer border-t border-outline transition-colors hover:bg-default"
                   >
                     <TableCell className="p-5">
```

Comment:
```suggestion
                    <TableCell className="p-4">
```
I think we can reduce the table row height a bit, looks cleaner and fits on the screen without scroll (14 inch screen)

Reply:
Before (p-5):

![image](https://github.com/user-attachments/assets/13b6e106-dee7-449a-b56e-8b1311ceed7b)


After (p-4): 

![image](https://github.com/user-attachments/assets/517e924c-f06f-45ed-b642-52794d0a4afe)